docs/cloud/azureai/consume-connections-on-cloud.md

```diff
@@ -0,0 +1,51 @@
+# Consume connections on cloud (Azure AI) in local
+
+For a smooth development flow that transitions from cloud (Azure AI) to local environments, you can directly utilize the connection already established on the cloud by setting the connection provider to "Azure AI connections".
+
+You can set the connection provider using the following steps:
+
+1. Navigate to the connection list in the VS Code primary sidebar.
+
+1. Click on the ... (more options icon) at the top and select the `Set connection provider` option.
+
+    ![img](../../media/cloud/consume-cloud-connections/set-connection-provider.png)
+
+1. Choose one of the "Azure AI connections" provider types that you wish to use. [Click to learn more about the differences between the connection providers](#different-connection-providers).
```

Comment:
we can change this [page](https://microsoft.github.io/promptflow/how-to-guides/set-global-configs.html#supported-configs) to link to our newly added page.

Reply:
Add a hint at the bottom.

By the way, I think this [page doc](https://microsoft.github.io/promptflow/how-to-guides/set-global-configs.html) is regarding to the CLI's feature points, and it would be better to include it in the CLI Reference(https://microsoft.github.io/promptflow/reference/pf-command-reference.html).

Reply:
we have added pf config cli reference

Reply:
The page you mentioned might display sdk usage also.